docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -855,6 +855,24 @@ Path to the default home dashboard. If this value is empty, then Grafana uses St
 On Linux, Grafana uses `/usr/share/grafana/public/dashboards/home.json` as the default home dashboard location.
 {{< /admonition >}}
 
+### `[k8s_dashboard_cleanup]`
```

Comment:
I think since this is behind a feature toggle for now, let's keep the doc changes out. 

Would you mind adding them to https://github.com/grafana/grafana/pull/101754? That is where we will have all the g12 docs around this feature when we enable it by default